setup.py

```diff
@@ -46,6 +46,7 @@
     'setuptools',
     'docutils',
     'sphinx',
+    'yowasp_yosys'
```

Comment:
Is there an "install_recommends" type thing?

Reply:
No, there is not. Nevertheless, since `yowasp_yosys` is the default option we should either add this dependency or make
system Yosys the default.